source/features/avoid-accidental-submissions.tsx

```diff
@@ -9,32 +10,48 @@ function addQuickSubmit(): void {
 }
 
 function onKeyDown(event: delegate.Event<KeyboardEvent, HTMLInputElement>): void {
+	const field = event.delegateTarget;
+	const form = field.form!;
 	if (
 		event.key !== 'Enter'
 		|| event.ctrlKey
 		|| event.metaKey
 		|| event.isComposing // #4323
-		|| select.exists('.suggester', event.delegateTarget.form!) // GitHub’s autocomplete dropdown
+		|| select.exists([
+			'.suggester', // GitHub’s autocomplete dropdown
+			'.rgh-avoid-accidental-submissions',
+		], form)
 	) {
 		return;
 	}
 
+	if (select.exists('.btn-primary[type="submit"]:disabled', form)) {
+		return;
+	}
+
+	const message = (
+		<p className="rgh-avoid-accidental-submissions my-1">
+			A submission via <kbd>enter</kbd> has been prevented. You can press <kbd>enter</kbd> again or use <kbd>ctrl</kbd>-<kbd>enter</kbd>.
```

Comment:
Alternative text suggestions welcome.

Also we need a Mac detection in `GitHub-helpers/index.ts`, like `navigator.userAgent.includes('Macintosh')`

```suggestion
			A submission via <kbd>enter</kbd> has been prevented. You can press <kbd>enter</kbd> again or use <kbd>{isMac ? 'cmd' : 'ctrl'}</kbd><kbd>enter</kbd>.
```